### PR TITLE
Rollback 0.23 to 0.16 fix

### PR DESCRIPTION
**Pull request by shmanu017**
## Description
Adding Deletion of Scheduler deployment box in current Rollback script from 0.23.14 to 0.16.15.
Ref: Zendesk [5619](https://astronomer.zendesk.com/agent/tickets/5619)

## 📋 Checklist

- [ ] The PR title is informative to the user experience

### CODE REVIEW

**Comment by kaxil** [comment]
There are some conflicts:

image

**Comment by kaxil** [comment]
I have changed the base to `release-0.23` - assuming we want to merge this in `release-0.23` branch, let me know if this is not what we want to do

**Review by danielhoherd — APPROVED** [review]

File: bin/migration-scripts/lts-to-lts/0.16-to-0.23/playbooks/tasks/rollback.yaml
Comment (vishwas-astro) [inline bin/migration-scripts/lts-to-lts/0.16-to-0.23/playbooks/tasks/rollback.yaml:241]: Typo - "all All"

File: bin/migration-scripts/lts-to-lts/0.16-to-0.23/playbooks/tasks/rollback.yaml
Comment (vishwas-astro) [inline bin/migration-scripts/lts-to-lts/0.16-to-0.23/playbooks/tasks/rollback.yaml:240]: @shmanu017 I feel this comment is not clearly describing what the intention of this section is. Can you take another stab at it please?

Comment (shmanu017) [reply bin/migration-scripts/lts-to-lts/0.16-to-0.23/playbooks/tasks/rollback.yaml]: @vishwas-astro Please review the new description

Comment (danielhoherd) [reply bin/migration-scripts/lts-to-lts/0.16-to-0.23/playbooks/tasks/rollback.yaml]: I think the old wording was better. How about:

```
# Delete scheduler deployment before rollback, in order to prevent the init container 'wait-for-airflow-migrations' from __________
```

**Review by kaxil — APPROVED** [review]

**Comment by vishwas-astro** [comment]
description looks good now @shmanu017 .. Thanks @danielhoherd

**Comment by vishwas-astro** [comment]
@shmanu017 / @danielhoherd We need to probably approve the LTS upgrader build here: https://app.circleci.com/pipelines/github/astronomer/astronomer/2990/workflows/d9690b5a-ef3d-4af1-8b6b-c546f018a92a after this merge.

@shmanu017 - can you test the rollback from 0.23.x to 0.16.y after this change?

**Comment by danielhoherd** [comment]
@vishwas-astro i approved the build just to see if tests pass, but we can wait for @aliotta's parallel fix to be tested and merged before releasing a new upgrader, unless there is some urgent need to get this out now.

**Comment by shmanu017** [comment]
@danielhoherd ,I accidentally pushed the new change to the release-0.23 branch directly.
Also, I tested its working fine during rollback it was working fine without any issues.
https://github.com/astronomer/astronomer/blob/f3df19e83a9aadb4e12b680108a5ea05e8a48d2b/bin/migration-scripts/lts-to-lts/0.16-to-0.23/playbooks/tasks/rollback.yaml#L241-L255

**Pull request merged**
